app/src/main/java/org/oppia/app/story/StoryActivityPresenter.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.app.story
+
+import androidx.appcompat.app.AppCompatActivity
+import org.oppia.app.R
+import org.oppia.app.activity.ActivityScope
+import org.oppia.domain.topic.TEST_STORY_ID_1
+import javax.inject.Inject
+
+/** The controller for [StoryActivity]. */
+@ActivityScope
+class StoryActivityPresenter @Inject constructor(private val activity: AppCompatActivity) {
+  fun handleOnCreate() {
+    activity.setContentView(R.layout.story_activity)
+    activity.supportActionBar?.setDisplayHomeAsUpEnabled(true)
+    activity.supportFragmentManager.beginTransaction().add(
```

Comment:
I suggest checking first that ``StoryFragment`` isn't created, otherwise this will recreate that fragment each time you rotate. This can be done via something like:

```kotlin
if (getStoryFragment() == null) {
  activity.supportFragmentManager.beginTransaction().add(...
}

...

private fun getStoryFragment(): StoryFragment? {
  return activity.supportFragmentManager.findFragmentById(R.id.story_fragment_placeholder) as StoryFragment
}
```